web/js/containers/embed.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button } from 'reactstrap';
+import googleTagManager from 'googleTagManager';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { getPermalink } from '../modules/link/util';
+import getSelectedDate from '../modules/date/selectors';
+import HoverTooltip from '../components/util/hover-tooltip';
+import history from '../main';
+
+class Embed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showOverlay: true,
+    };
+    this.clickOverlay = this.clickOverlay.bind(this);
+    this.renderEmbedLinkBtn = this.renderEmbedLinkBtn.bind(this);
+    this.newTabLink = this.newTabLink.bind(this);
+  }
+
+  clickOverlay() {
+    this.setState({ showOverlay: false });
+  }
+
+  newTabLink() {
+    const { selectedDate } = this.props;
+    const queryString = history.location.search || '';
+    const permalink = getPermalink(queryString, selectedDate);
+    googleTagManager.pushEvent({
+      event: 'embed_open_new_tab',
+    });
+    window.open(permalink, '_blank');
+  }
+
+  renderEmbedLinkBtn() {
+    const { isMobile } = this.props;
+    const buttonId = 'wv-embed-link-button';
+    const labelText = 'Open new tab with content in Worldview';
```

Comment:
This tool tip could use some clarification... something like, "Open this Worldview map in a new window".